### ISSUE TITLE
[CTW/8x5] Ruediger's Octawool

### ISSUE THREAD

**Issue by RuedigerLP** [OvercastCommunity/CommunityMaps#201]
### Checklist
- [x] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map
- [x] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the README
- [x] I have created an XML file
- [x] I have created an map image
- [x] I have uploaded the map zip file to a file sharing service
- [x] The map has been tested locally to make sure it works

# Map Name
Ruediger's Octawool

## Gamemode & Map Description  
Octawool is an 8-team ctw built in march 2019 for the 2nd MMC. Currently it has 5 players per team, i'm pretty sure 8 per would be possible.

## Team Sizes
8 teams á 5 players

https://github.com/OvercastCommunity/CommunityMaps/pull/200

**Comment by calcastor** [comment]
Just tested this and I would personally say it plays pretty well! Tested with full team sizes though we bumped to 7 on each team due to the amount of online players at the time just in case. Some early feedback I have is regarding the buffs teams have in their own base regions. While speed and resistance are fine, I would say that both the strength and anti-knockback are unnecessary. The anti-knockback should at the very least be reduced, and I think that only one of resistance or strength should be provided, though only providing resistance seems significantly more balanced.
